Fixed errand filters

MaxErikssonSK opened the pull request:
Added filter base component to use in both desktop and mobile. Removed comboboxes.
Added colors for prio filter.

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix
- [x] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Content/Data

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [x] No
      
## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly (if applicable).
- [x] I have added/updated tests to cover my changes (if applicable).

MaxErikssonSK committed:
Fixed errand filters

Added filter base component to use in both desktop and mobile.
Removed comboboxes.
Added colors for prio filter.

OliverBorgstrom commented on frontend/src/components/filtering/desktopFiltering/errand-filter-base.component.tsx:
```diff
@@ -0,0 +1,54 @@
+export const CasedataFilterBase: React.FC<Props> = ({
+  ownerFilterHandler,
+  ownerFilter = true,
+  backgroundColor = 'primary-50',
```
Varför sätter vi backgroundColor som en prop? Utöver det kan det inte vara en primary color

MaxErikssonSK replied:
Hade en tanke att bakgrundsfärgen kunde vara lämplig att välja då denna komponent kan användas på fler ställen. I vårt fall ska det vara olika färger för desktop och mobil. Finns säkert bättre sätt att lösa det på.

MaxErikssonSK committed:
Removed prop for color

Moved out design to make the code easy to follow.
Made some changes regarding mobile adaptation.
